app/assets/components/messages/src/ui/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
Can remove, unless we plan to develop in isolation. Again, this may not be possible.